src/Interface/InterfaceView.css

```diff
@@ -1,4 +1,5 @@
-/* ContactPeopleView.css */
+@import "../Utils/wrapValue.css";
```

Comment:
css from utils?

Reply:
any suggestions?)

Reply:
common folder :)

Reply:
so `src/common/styles/wrapValue.css` is fine?